docs/journalist/sources.rst

```diff
@@ -16,6 +16,12 @@ Once logged in, you will see a chat-like user interface consisting of three pane
   with a specific source (messages, files, and journalist replies) will be
   displayed here.
 
+- The top panel holds a toolbar that allows you to apply actions in bulk.
```

Comment:
Wondering if "bulk" is a meaningful term here. I feel like if I were to encounter this organically, it might not make sense to me. It's also the only time the term is used here in the docs.

Maybe?:

```suggestion
- The top panel holds a toolbar that allows you to apply actions to multiple items at once.
```